students/jakielczyk_miroslaw/lesson_09_reading_and_writing_files/mad_libs.py

```diff
@@ -0,0 +1,52 @@
+file = open('file_to_be_modified.txt', 'r')
```

Comment:
OK, but The preffered way to operate on files is using context managers: `with` 
Check out : https://stackoverflow.com/questions/9282967/how-to-open-a-file-using-the-open-with-statement